samples/Azure.Management.Storage/Generated/RestOperations/BlobContainersRestOperations.cs

```diff
@@ -112,10 +99,9 @@ public async Task<Response<ListContainerItems>> ListAsync(string subscriptionId,
                         return Response.FromValue(value, message.Response);
                     }
                 default:
-                    throw await ClientDiagnostics.CreateRequestFailedExceptionAsync(message.Response).ConfigureAwait(false);
+                    throw new RequestFailedException(message.Response);
             }
         }
-
```

Comment:
Was it intentional to remove this line break?  I like it for readability.

Reply:
No, accidental. Thanks for noticing!

Reply:
Fixed